ui/css/itcss/components/newui-sections.scss

```diff
@@ -50,14 +50,6 @@
 // main-container media queries
 
 @include design-system.screen-sm-min {
-  .lap-visible {
-    display: flex;
-  }
-
-  .phone-visible {
-    display: none;
-  }
```

Comment:
Not used

![Screenshot 2025-02-07 at 11 54 51 AM](https://github.com/user-attachments/assets/d8af8a73-b2cd-47d9-969f-ed3b0a087a8d)